ffi_tests_output.txt

```diff
@@ -406,31 +381,29 @@ Passed: tests/ffi/vmspecific_function_callbacks_test.dart
 Compiling tests/ffi/vmspecific_function_gc_test.dart
 Running tests/ffi/vmspecific_function_gc_test.dart
 
-../../runtime/vm/simulator_arm64.cc: 955: error: expected: instr == NULL || reg != R18
-version=3.0.6 (stable) (Fri Jul 21 14:08:38 2023 -0700) on "macos_simarm64"
-pid=51474, thread=15875, isolate_group=main(0x13d812200), isolate=main(0x13d81f400)
+Simulator hit Unimplemented instruction: at 0x102c25b7c, last_pc=0x102dcfb9c
```

Comment:
Ended up in clang code?

Reply:
Yes, it looks like it.  We are calling native code through the ExitSafepoint stub.  We either need to ensure there's a "to CPU" transition in that stub, or that the stub itself is run on the CPU.